src/stdlib_strings.f90

```diff
@@ -93,6 +93,17 @@ module stdlib_strings
         module procedure :: replace_all_char_char_char
     end interface replace_all
 
+    !> Returns the number of times substring 'pattern' has appeared in the
+    !> input string 'string'
+    !> [Specifications](../page/specs/stdlib_strings.html#count)
+    !> Version: experimental
```

Comment:
Version can be considered as meta-data by ford (https://github.com/Fortran-FOSS-Programmers/ford/wiki/Documentation-Meta-Data)
```suggestion
    !> Version: experimental
    !>
    !> Returns the number of times substring 'pattern' has appeared in the
    !> input string 'string'
    !> [Specifications](../page/specs/stdlib_strings.html#count)
```